CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ### vNEXT
 
 - Improve identity template literal tag docs. [PR #254](https://github.com/apollographql/eslint-plugin-graphql/pull/254) by [Jayden Seric](https://github.com/jaydenseric).
+- Add support for GraphQL 15. [PR #271](https://github.com/apollographql/eslint-plugin-graphql/pull/271) by [Scott Taylor](https://github.com/staylor).
+- Update all `devDependencies` - upgrades the project to use Babel 7 and ESLint 6. [PR #271](https://github.com/apollographql/eslint-plugin-graphql/pull/271) by [Scott Taylor](https://github.com/staylor).
+- **BREAKING**: Minimum supported Node.js version is now Node.js 10; Dropped support for Node.js 8. [PR #271](https://github.com/apollographql/eslint-plugin-graphql/pull/271) by [Scott Taylor](https://github.com/staylor).
```

Comment:
I explicitly made a call-out to dropping Node.js from `engines`. Even if the relatively unenforced `engines` was the only thing preventing it from working on new versions, I think it's okay for us to take a stance on it 3ecf7deeeeeb2a7e87770204631f03a0b72cf263 😄 .